src/components/SettingsModal.tsx

```diff
@@ -455,7 +455,6 @@ export const SettingsButton = memo(() => {
       <SettingsModal
         isOpen={isSettingsOpen}
         onClose={onSettingsClose}
-        onOpen={onSettingsOpen}
```

Comment:
Why did you get rid of this?

Reply:
Because `SettingsModal` has no `onOpen` property. Look at the props at the start of the file.

Reply:
Oh haha. Guess I just missed that when I refactored it into its own file